mend_sca_cleanup_tool/sca_cleanup_tool.py

```diff
@@ -116,10 +133,15 @@ def delete_scan(product_token, project):
                 "projectToken": project['token'],
                 "agentInfo": AGENT_INFO
             })
-    response_obj = json.loads(post_api_request(request).decode("utf-8"))
+    r = post_api_request(request)
```

Comment:
Single letter variable, I would expand this out.

Reply:
We can keep it. And create new PSD for improvement this tool, for example

Reply:
The point of the code review is to catch code quality concerns in addition to functionality. We should clean it up while we can instead of kicking the code down the road.